### ISSUE TITLE
How to not parse a nested pyproject.toml?

### ISSUE THREAD

**Issue by gbolmier** [charliermarsh/ruff#1812]
I love this project, keep up the good work ❤️ 

```sh
$ ruff --version
ruff 0.0.218
$ python --version
Python 3.8.15
```

I'm developing a Cookiecutter template which gives me the following tree structure:

```
.
├── ...
├── pyproject.toml
└── {{ cookiecutter.repo_name }}
    ├── pyproject.toml
    └── ...
```

where the nested `pyproject.toml` has Jinja2 placeholders causing parsing errors:

```sh
$ ruff .
error TOML parse error at line 26, column 2
   |
26 | {% if cookiecutter.use_mypy == "Yes" %}
   |  ^
Invalid inline table
Expected `}`
```

Using the `config` flag to only use the root level `pyproject.toml` doesn’t prevent the nested `pyproject.toml` to be parsed. Is there a way to not parse it?

**Comment by charliermarsh** [comment]
Ahh interesting. Yeah, we should fix this. I have to look back at the code. We’re probably parsing it out of convenience (due to some specific details related to the abstractions).

**Comment by charliermarsh** [comment]
(This should be fixed in the next release (v0.0.220), which will go out later today. Would appreciate if you could test and confirm once it's live!)

**Comment by gbolmier** [comment]
Thank you so much for the blazing fast fix @charliermarsh 🙏
`ruff --config=pyproject.toml .` now works like a charm 🚀
